src/extension.ts

```diff
@@ -66,6 +66,7 @@ export let extensionManager: ExtensionManager | null = null;
 
 type RunCMakeCommand = (project: CMakeProject) => Thenable<any>;
 type QueryCMakeProject = (project: CMakeProject) => Thenable<string | string[] | null>;
+type FolderTargetNameArgsType = vscode.WorkspaceFolder | string | { folder?: vscode.WorkspaceFolder | string; targetName?: string };
```

Comment:
```suggestion
type FolderTargetNamedArgsType = vscode.WorkspaceFolder | string | { folder?: vscode.WorkspaceFolder | string; targetName?: string };
```

Reply:
It means this type contains `Folder` and `TargetName`. It's not meant to be an adjective. But I'm open to better naming.

Reply:
Ahh, that seems reasonable to me, I just read it wrong.